tests/test_reservation.py

```diff
@@ -1,4 +1,5 @@
 from datetime import datetime
+from http import HTTPStatus
 from timeless.restaurants.models import ReservationStatus, Reservation
```

Comment:
@carlosmiranda according to pep8 here should be blank line between 2 and 3 lines